client/src/components/web-extension/ImportWallet/ImportWallet.scss

```diff
@@ -60,7 +59,10 @@
 		}
 
 		&--body {
+			display: flex;
```

Comment:
Adding `display: flex` here breaks layout in some common pages. 

<img width="373" alt="image" src="https://user-images.githubusercontent.com/610234/198308006-fc543a44-48ea-4b7a-ba2a-5d7c3bb57443.png">